### ISSUE TITLE
This package appears unmaintained

### ISSUE THREAD

**Issue by benogle** [atom-community/language-scala#28]
We plan create a fork at https://github.com/atom-community, and to point the Atom [package listing entry](https://atom.io/packages/language-scala) to this fork on __June 1st__ provided no interaction on this thread. Other options:

* Fix deprecations on this fork
* Transfer ownership of the repo to @cjwilburn

Thanks!

**Comment by laughedelic** [comment]
Hurray! :+1:

**Comment by petewalker** [comment]
Excellent... my last pre-1.0 deprecation warning!

**Comment by petewalker** [comment]
@benogle is this still going ahead?

**Comment by mcous** [comment]
Per @lierdakil over in #27 there may be some holdup getting this squared away? For what it's worth I'm doing a lot of Scala for my work job these days, and Atom is my editor of choice, so I'd be happy to maintain this package if that would help

**Comment by cjwilburn** [comment]
I'm still interesting in taking it over if no one else is stepping forward.

**Comment by jroesch** [comment]
Hey everyone, sorry about being dark for the past couple of months I've had a lot going on in my life and haven't been spending much time on GitHub. I'm happy to have other contributors join and am better reachable by email.  I'll merge everything up today.

**Comment by mcous** [comment]
:+1:

**Comment by benogle** [comment]
@jroesch can you transfer ownership to my github account? I can place it in the https://github.com/atom-community org, and give everyone interested here access. That way if someone goes MIA, someone can always push to it.

**Comment by btd** [comment]
That will be awesome. :+1:

**Comment by raboof** [comment]
:+1:

**Comment by chandu0101** [comment]
:+1:

**Comment by GitsMcGee** [comment]
:+1:

**Comment by cjwilburn** [comment]
@benogle are you still moving ahead with moving this over to atom-community? There's an outstanding pull request #30 that would be nice to have merged in the near future.

**Comment by gangstead** [comment]
I talked with @jroesch via twitter and we agree moving it over to atom-community is the best way to move forward.  I'd do it right away but as a contributor (not an owner) I don't have the rights, so we're waiting on him.

Re: PR #30 I didn't merge it in because I don't think it's ready.  The var itself needs to have different styling, not the keyword `var`.

**Comment by cjwilburn** [comment]
Awesome. Yeah, I hadn't looked at #30 in a bit, and didn't realize that the outstanding issues hadn't been rectified. But, thanks for getting @jroesch to move it over. That's huge.

**Comment by benogle** [comment]
@jroesch needs to transfer the repo to my account, and I will put into the atom-community org. I emailed him about this just now. If anyone has a better communication channel with him (@gangstead?), please ask him to transfer it to my account.

**Comment by benogle** [comment]
Ok, this has been transferred. @cjwilburn and @gangstead have access. I cant remember who else should be a committer. Let me know

**Comment by benogle** [comment]
Thanks @jroesch

**Comment by cjwilburn** [comment]
Thanks @jroesch and @benogle

**Comment by mcous** [comment]
Hey @benogle, despite this package being transferred to atom-community, it still seems pretty unmaintained. @gangstead has mentioned (over in #33) that he no longer has write access, and I don't know about @cjwilburn. Do you know who at atom-community has the permissions to fix this?

I'm also happy to help maintain this package, if you know who I can talk to about that.

**Comment by benogle** [comment]
@mcous you have been invited.

**Comment by mcous** [comment]
Much appreciated! Thanks for getting this sorted out
